worker/stripe_charge_succeeded.js

```diff
@@ -11,10 +30,18 @@ module.exports = function(notifier_messager, mailroom) {
     }
 
     var currency_code = event.data.currency.toUpperCase();
+    // If this is a zero decimal currency then use it directly
+    // Otherwise divide by 100 to get currency major.minor amount
+    var amount;
```

Comment:
if you default this to `var amount = event.data.amount` then you can have one if with no else.